classes/class-query.php

```diff
@@ -118,12 +118,10 @@ public function query( $args ) {
 
 				$field = str_replace( array( 'record_', '__in' ), '', $key );
 				$field = empty( $field ) ? 'ID' : $field;
-				$type  = is_numeric( array_shift( $value ) ) ? '%d' : '%s';
+				$type  = is_numeric( $value[ 0 ] ) ? '%d' : '%s';
 
-				if ( ! empty( $value ) ) {
-					$format = '(' . join( ',', array_fill( 0, count( $value ), $type ) ) . ')';
-					$where .= $wpdb->prepare( " AND $wpdb->stream.%s IN {$format}", $field, $value ); // @codingStandardsIgnoreLine prepare okay
```

Comment:
`$wpdb->stream.%s` always becomes `$wpdb->stream.'FIELDNAME'` which never works. We need to specify the field name picking it from a list of known "property" field names and use it directly as `$wpdb->stream.{$field}`.